### DISCUSSION TITLE
Cancellation of pro subscription

### DISCUSSION THREAD

**Discussion by Swarnim37** [vercel/community#1105] [Help] [answered]
Hello,  I would like to cancel my pro subscription. I have already requested twice at support 5 days ago . I tried to find but couldn't find anything regarding cancellation of the pro subscription other that contacting support.

Please let me know if there is any other way, I don't want to pay for another month.

Thank you in advance.

**Comment by swarnava** [comment]
Hi,

Could you share the ticket #?

**Reply by Swarnim37** [reply]
Hi,
Case #00109997 first
Case #00110412 second 

Thank you in advance.

**Reply by amyegan** [reply]
I see that Swarnava replied on your ticket, but I want to also share the answer here in case anyone else needs help with this topic. 

To cancel your Pro subscription, you will need to delete your Vercel Team. Go to your Team's Dashboard, then open the Settings tab. Near the bottom of the General Settings you will find a card titled `Delete Your Team` which will let you permanently delete the team, effectively cancelling your subscription.

**Reply by manaberg** [reply]
I wanted to cancel my subscription but I still want to use the month that I paid for, is there a way to do that?

**Reply by swarnava** [reply]
Unfortunately no, you need to delete the team in order to cancel the subscription. You can delete the team before your last billing date

**Reply by jbccollins** [reply]
This is not a great solution. I would like to pause my pro benefits for a while and then start then back up when I need them again. Please consider allowing for such a use case. My "team" is just me.

**Reply by rijk** [reply]
This is definitely not great, to say the least. Recreating a team is a lot of work, especially when there's KV/blob stores linked to it. I get that you may want to discourage people to downgrade, but in the end I find this customer-unfriendly.

**Reply by Rafath-Auvee** [reply]
Will they refund the money after I cancel my vercel pro plan?

**Comment by wuzhuzhu** [comment]
I have same issue,  case number is 00164245.
it is no possible way to cancel a pro plan.
submitted a ticket and 2 days gone, no one cares the ticket.

**Reply by swarnava** [reply]
To cancel your Pro subscription, you will need to delete your Vercel Team. Go to your Team's Dashboard, then open the Settings tab. Near the bottom of the General Settings you will find a card titled `Delete Your Team` which will let you permanently delete the team, effectively cancelling your subscription.

**Reply by gauravz7** [reply]
Will you refund the subscription amount ?

**Reply by atakkant** [reply]
This solves the isuue but be careful, once you delete your team, you will lose all the project as well

**Reply by amyegan** [reply]
It is no longer necessary to delete a team if you don't need a Pro plan anymore. You can downgrade to hobby in your team's billing settings – https://vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby

If you do want to delete a team, be sure to transfer all projects and domains before deleting the account to ensure there are no disruptions to your deployments.

**Comment by amyegan** [comment, marked as answer]
I know this post is a bit old now, but I wanted to share that a new solution is available. We've improved the process of downgrading a team from pro to hobby. You'll now find a "**Downgrade Plan**" button on your team's billing settings. 

Full steps and more information can be found at [vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby](https://vercel.com/docs/accounts/plans/hobby#downgrading-to-hobby)

**Reply by vsimplify** [reply]
Will you provide a refund if I downgrade the same day as you charged me?

**Reply by paulwongx** [reply]
@amyegan I clicked on the pro trial to see the UI and now I can't go back to the hobby plan as there's no downgrade plan button during the trial. Will I be automatically billed after the trial period? I can't even cancel the trial currently.

**Reply by amyegan** [reply]
At the end of the trial your team will automatically [revert to a Hobby plan](https://vercel.com/docs/accounts/plans/pro/trials#revert-to-a-hobby-plan) if you have not added a payment method. If you do have a payment method then the team will automatically [upgrade to a paid Pro plan](https://vercel.com/docs/accounts/plans/pro/trials#upgrade-to-a-paid-pro-plan).

https://vercel.com/docs/accounts/plans/pro/trials#post-trial-decision

**Reply by rosaldo260** [reply]
Why is the PRO plan immediately cancelled and returned to Hobby plan if a month of PRO plan was paid.

That's just trying to prevent users from downgrading.

[a comment by an automated account is not shown]
